recipe/meta.yaml

```
@@ -30,10 +30,79 @@ source:
       - patches/Improve-logic-for-finding-the-macos-sysroot-for-cond.patch  # [osx]
       - patches/0008-Remove-trailing-null-bytes-from-the-sysroot-that-com.patch       # [variant and variant.startswith("root_")]
       #- patches/root/0001-Attempt-to-extract-clang-patches-from-ROOT.patch  # [variant == "root"]
-      - patches/root/0001-ROOT-include-files.patch
-      - patches/root/0002-ROOT-libs-1.patch
-      - patches/root/0003-ROOT-libs-2.patch
-      - patches/root/0004-ROOT-libs-3.patch
+      - patches/root/0001-Temporary-parsers-should-not-manipulate-global-parse.patch
```

These should all end with "selectors" to make it so the lines are only applied if it's building the root variant.

"selectors" are basically special comments of the form `# [xxxx]` where `xxxx` is a python-like expression. If `xxxx` evaluates to false the line is removed.

```suggestion
      - patches/root/0001-Temporary-parsers-should-not-manipulate-global-parse.patch # [variant and variant.startswith("root_")]
```